src/python/grpcio/grpc/_channel.py

```diff
@@ -1439,15 +1588,19 @@ def _unsubscribe(state, callback):
                 break
 
 
-def _augment_options(base_options, compression):
+def _augment_options(
+        base_options: Optional[ChannelArgumentType],
```

Comment:
Why `Optional`?

Reply:
On second thought, we don't need `Optional` here, will remove it. (this comes from the 2nd return value of `_separate_channel_options`, it can be an empty list but not `None`)